engli/110_jumle.md

```diff
@@ -7,74 +7,72 @@ They may consist of only one word, which draws the listener's attention.
 
 **mau!**
 – A cat!  
-**barxa!**
+**xava!**
 – (It) rains!
 
 ## Stative Sentences
 
 ### With noun or pronoun subject
 
 Normally a sentence consists of a subject and a predicate.
-In the simplest sentences, the subject is a noun or a pronoun and the predicate is an adjective or a noun.
+In the simplest sentences, the subject is a pronoun and the predicate is an adjective or a noun.
 
-**me boni.**
+**me bon.**
 – I am good.  
-**le novi.**
+**le nove.**
 – It is new.  
 **le Sara.**
-– She is Sara.  
-**Sara boni.**
+– She is Sara.
+
+When the subject is a noun, the connecting verb **si** (_to be_) is necessary.
```

Comment:
Why? And Chinese and many creole languages, one doesn't use a copula in such cases.

Reply:
This is one of those structures where the word class markers did really well. There is need to structurally differentiate things like "crystal-clear" and "crystal is clear".

What Chinese does is that it inserts a degree adverb between the subject and the adjective: wǒ māma hěn hǎo. Pandunia could use that structure: mo de mama tre bon. Sara tre bon. kristal tre klar. idea tre kristal klar.